tutorials/foreign-object.html

```diff
@@ -0,0 +1,813 @@
+            <code>input</code>, and then have to access DOM elements themselves to update the <code>'value'</code> property. If using 
+            <code>foreignObject</code>, users should be able to set the <code>'value'</code> through the model at any time they want. It's for this
+            exact reason that we created the special <code>props</code> attribute, and it can be utilized as follows:
+        </p>
+
+        <pre><code>// Example markup
+&lt;input @selector="example" type="text" value="initial" /&gt;
+
+...
+
+// Set the input value on the model
+element.attr('example/props/value', 'current');
+
+...
+
+// For illustration purposes - not required to access DOM elements in this manner
+const input = document.querySelector('input');
+
+// Expected values
+console.log(input.value); // current (JointJS sets the 'value' DOM property internally)
+console.log(input.defaultValue); // initial
+console.log(input.getAttribute('value')); // initial
+</code></pre>
+
+        <p>
+            The supported properties when using the <code>props</code> attribute are:
+        </p>
+
+        <ul>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#checked" target="_blank"><code>checked</code></a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option#attributes" target="_blank"><code>selected</code></a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#disabled" target="_blank"><code>disabled</code></a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#readonly" target="_blank"><code>readOnly</code></a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/contenteditable" target="_blank"><code>contentEditable</code></a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#value" target="_blank"><code>value</code></a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#indeterminate_state_checkboxes" target="_blank"><code>indeterminate</code></a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#attributes" target="_blank"><code>multiple</code></a></li>
+        </ul>
+
+        <br>
+
+        <h3>Caveats!</h3>
+
+        <p>
+            While all that we have covered so far is great for users of JointJS, even in this day and age, it's not without some caveats.
+            Firstly, we will cover some minor points regarding syntax that you should be aware of when using HTML within your <code>markup</code>.
+            Afterwards, we'll look at some browser related quirks which mostly centre around Safari.
+        </p>
+
+
+        <h4>Syntax</h4>
+
+
+        <h4 style="font-weight: normal;">Boolean Attributes</h4>
```

Comment:
should be h5 (to make the structure of the page clear to screen readers and such) - but you can define the style to look the same as this